backend/post-service/src/test/java/com/speakapp/postservice/PostServiceApplicationTests.java

```diff
@@ -1,13 +1,55 @@
 package com.speakapp.postservice;
 
+import com.speakapp.postservice.dtos.*;
+import com.speakapp.postservice.entities.*;
+
+import com.speakapp.postservice.mappers.PostMapper;
+import com.speakapp.postservice.repositories.CommentReactionRepository;
+import com.speakapp.postservice.repositories.CommentRepository;
+import com.speakapp.postservice.repositories.PostReactionRepository;
+import com.speakapp.postservice.repositories.PostRepository;
+import com.speakapp.postservice.services.PostService;
 import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
 
+
+import java.util.List;
+import java.util.UUID;
+
+
 @SpringBootTest
 class PostServiceApplicationTests {
 
+	@Autowired
```

Comment:
Jak zrobiłeś testy to też je wrzuć śmiało

Reply:
Coś próbowałem napisać, żeby sprawdzić poprawne działanie, ale ostatecznie nic z tego konkretnego nie wyszło, także wgl miałem tego nie komitować i zostawiłem troszkę zaśmiecone widzę, także to ogarnę w komicie